docs/tasks/debug-application-cluster/audit.md

```diff
@@ -9,7 +9,7 @@ title: Auditing
 * TOC
 {:toc}
```

Comment:
Maybe add the beta label, since you're removing the legacy section?

```
{% include feature-state-beta.md %}
```

Reply:
Done